fix 1.20.1 not compiling due to cccompatibility

cylian914 opened the pull request:
error was: ```error: Unable to locate obfuscation mapping for @WrapOperation target getRange```

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ValkyrienSkies/Valkyrien-Skies-2?pullRequest=1083) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ValkyrienSkies/Valkyrien-Skies-2?pullRequest=1083) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ValkyrienSkies/Valkyrien-Skies-2?pullRequest=1083) it.</sub>

blockninja124 commented:
I haven't tested this specifically, but the change looks good to me. 

(I doubt the change could break anything, and it probably _is_ needed since we don't want to remap for CC mixins)

blockninja124 reviewed (dismissed):
I haven't tested this specifically, but it looks good to me.

(I doubt the change could break anything, and it probably is needed since we don't want to remap for CC mixins)

cylian914 committed:
rename afterFluidStateUpdate to inShipFluidPush

cylian914 committed:
Remade the in ship fluid flow push

The previous way was overriding the function (and copying vanilla
behaviour to apply it to ship) which isn't great for
compatibility with other mod that mixin in this function

this uses more complex mixin to let vanilla/forge gather all the fluid that
will push the entity then inject code that recall the function and
switching variable to make the gathering of ship (reuseing the
function make it more compatible with other mod's mixin) and combining
with vanilla then letting the vanilla/forge way of managing fluid
pushing

cylian914 commented:
i completely forgot i made this is a while ago
its already fixed iirc so im closing the pr
